src/lib/permutations/ascon_perm/info.txt

```diff
@@ -0,0 +1,8 @@
+<internal_defines>
+ASCON_PERM -> 20250816
```

Comment:
No need for this I think - all users just hard depend on it rather than checking for the macro